### DISCUSSION TITLE
Confusion matrix in Quality Estimation

### DISCUSSION THREAD

**Discussion by bittlingmayer** [machinetranslate/machinetranslate.org#254] [General]
In [*Outbound Translation User Interface Ptakopět: A Pilot Study*](https://arxiv.org/abs/1911.10835), it mentions *false positives*.

Is a "positive" a good translation or a *bad* translation (like testing positive for a disease in a medical context)?

@zouharvi I assume your setup is in harmony with that in [Lucia Specia's book](https://www.morganclaypool.com/doi/abs/10.2200/S00854ED1V01Y201805HLT039) on QE.

**Comment by zouharvi** [comment]
Negatives (in terms of QE classification) are segments/words/tokens classified as incorrect.

- True negative: segment is incorrect and is reported incorrect
- True positive: segment is correct and is reported as correct
- False negative: segment is correct but is reported as incorrect
- False positive: segment is incorrect but is reported as correct

**Comment by zouharvi** [comment]
An example: _The kids fell asleep ._

|Reference|Translated|QE|Classification|
|-|-|-|-|
|Die|Die|OK|TP|
|Kinder|Kinder|BAD|FN|
|sind|ist|OK|FT|
|eingeschlafen|eingeschlufen|BAD|TN|
|.|.|OK|TP|

**Comment by bittlingmayer** [comment]
Thanks!

At ModelFront, we've been doing the opposite!

**Comment by zouharvi** [comment]
@bittlingmayer Can this issue be closed?

**Comment by cefoo** [comment]
I think this is better fitted to be stored under [Discussions](https://github.com/machinetranslate/machinetranslate.org/discussions) instead of closing an issue?

I may be wrong, I'm still new to GitHub! 🙃

**Comment by bittlingmayer** [comment]
Sure!
